worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala

```diff
@@ -128,12 +131,12 @@ class PushDataHandler extends BaseMessageHandler with Logging {
     val isMaster = mode == PartitionLocation.Mode.MASTER
 
     // For test
-    if (isMaster && conf.testPushMasterDataTimeout && !pushMasterDataTimeoutTested) {
+    if (isMaster && !pushMasterDataTimeoutTested) {
```

Comment:
make sense

Reply:
Apologies for the confusion before. In fact, `conf.testPushMasterDataTimeout` will be invoked every time the condition is checked. This is because, in the production environment, `!pushMasterDataTimeoutTested` always returns `true`. Simply reordering the conditions will not resolve this issue.

Reply:
Right, thanks for your clearance :)